### PR TITLE
👐🏽 Set up pagination in frontend

### PR DESCRIPTION
**Pull request by nkonjeti**
## Description
Set up pagination in frontend and link it to backend 

Fixes #<number>

## Screenshots
Still a work in progress
Screen Shot 2021-04-08 at 12 56 21 AM

### CODE REVIEW

File: client/src/pages/Resources.jsx [hunk 1]
```diff
@@ -164,6 +168,12 @@ function Resources({
     setFilteredResources(
       newResources == null ? [] : newResources.result.totalData,
     );
+    setResourceCount(
+      newResources == null
+        ? []
+        : newResources.result.totalCount[0].resourceCount,
+    );
```
Comment (jackieo5023) [inline client/src/pages/Resources.jsx:175]: Since `resourceCount` is a integer, if `newResources` is `null` maybe we can set it to 0 instead?

**Review by jackieo5023 — APPROVED** [review]
AWESOME!!!! I think after addressing my one comment and resolving the merge conflicts we should be good to go!

File: client/src/components/ResourcesGrid.jsx [hunk 2]
```diff
@@ -6,19 +6,37 @@ import ResourcePreview from './ResourcePreview';
...
-        xl: 6,
+        sm: 3,
+        md: 3,
+        lg: 3,
+        xl: 3,
         xxl: 3,
       }}
       dataSource={filteredResources}
+      pagination={{
+        pageSizeOptions: ['6', '15', '30'],
```
Comment (Feeshay) [inline client/src/components/ResourcesGrid.jsx:32]: (said this before last dev night)
Is it possible to change this to ints so we don't need the parseInts in the mongodb stuff? No big deal if you can't

**Review by Feeshay — APPROVED** [review]

File: client/src/components/ResourcesGrid.jsx (hunk 2 above)
Comment (Feeshay) [reply client/src/components/ResourcesGrid.jsx:32]: also remember fix the linting (run `yarn format`) and conflicts

**Comment by jackieo5023** [comment]
Ah... I forgot about this. We have a component `SavedResources` that also uses the `ResourceGrid` - would you be able to pass down the components to `ResourcesGrid` from `SavedResources` that are necessary? The API call is in the same component I believe, so it shouldn't be difficult. The only thing is we currently call `getResources` where in `Resources.jsx` we call `getResourcesByCategory`. We can now use `getResourcesByCategory`, but pass in null values for everything but page and size.

File: client/src/components/ResourcesGrid.jsx (hunk 2 above)
Comment (nkonjeti) [reply client/src/components/ResourcesGrid.jsx:32]: thanks Luciana! Also sadly I don't think I can change it to ints because antd wants a string array.

**Pull request merged**